example/empty.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Example Scene</title>
+    <script src="js/aframe.min.js"></script>
+  </head>
+  <body>
+    <a-scene stats>
+    </a-scene>
+    <script src="http://localhost:3333/build/aframe-inspector.js"></script>
```

Comment:
Yes, many times I want to try the inspector with an empty scene and it's annoying to edit the current example or delete every element.